ILSpy/LoadedAssembly.cs

```diff
@@ -142,13 +144,12 @@ private void LoadSymbols(ModuleDefinition module)
 			if (!module.HasDebugHeader) {
 				return;
 			}
-			byte[] headerBytes;
-			var debugHeader = module.GetDebugHeader(out headerBytes);
-			if (debugHeader.Type != 2) {
+			var debugHeader = module.GetDebugHeader().Entries.FirstOrDefault();
```

Comment:
Question: would it be better to just use Cecil's default symbol reader provider here [like mine did](https://github.com/airbreather/ILSpy/commit/4d18cd33e29de480f172b0b23aa4a1fefab845fa#diff-e8580605d35a71ed7bf7fce6a58b0198R142)?  I saw a lot of the same-looking code inside the implementation of that (I think it would make this a fix for #789)... I'm not sure if that's related to the bigger file-lock issue I'd commented on before starting to go through your actual code to compare with what I'd done.